src/main/resources/config/application.yml

```diff
@@ -1,5 +1,10 @@
 faf-api:
   version: #faf-api.version#
+  map:
+    final-directory: ${MAP_UPLOAD_PATH}
+    map-preview-path-large: ${MAP_PREVIEW_PATH}
+    large-previews-url-format: ${MAP_PREVIEW_PATH}
```

Comment:
The application won't start w/o these variables being set. A developer should not be forced to set any values.
To force them being set, move this config to `application-prod.yml` and use the same thing in `application-dev.yml` but with sensible default values (e.g. `./build/....`)